CONTRIBUTING.md

```diff
@@ -227,24 +227,44 @@ To start debugging JHipster with **VSCode**, open the generator code in your wor
 
-## Local Travis Build
+## Local Travis Build and explore application generated
 
-You can run the travis builds locally by following below commands
+You can run the Travis builds locally by following below commands, and track rendered pages.
 
-CD into the travis folder `cd travis` from the generator source code root folder
+#### How to use it
 
-Run `./build-samples.sh [command_name] [sample_name:optional]`
+1. Change directory into the [./travis](./travis) folder `cd travis` from the generator source code root folder
 
-This will create the travis sample project under the `travis/samples` folder with folder name `[sample_name]-sample`. You can open this application in your editor or IDE to check it further. You can also run tests locally on the project to verify
+2. Do not forget to run `yarn link`.
 
-Sample name is optional and can be any of the folder name in the `travis/samples` folder. If not specified the it will mean all samples
+3. Run `./build-samples.sh generate|generateandtest [ sample_name[,sample_name][,...] ] [ --colorizelogfile ] ` (parameters between brackets are optionals).
 
-Command name can be as below
+    * `generate` and `generateandtest` create the Travis sample project under the `travis/samples` folder with folder name `[sample_name]-sample`. You can open this application in your Text Editor or IDE to check it further. You can also run tests locally on the project to verify.
+        * By default, run on all samples. Choose those you want in `./build-samples.sh help`.
+        * You could build several samples in same time by separated it by comma (e.g. `ngx-default,react-default`).
+        * `--colorizelogfile`: keep color in your log file, **this optional parameter is strongly advise for readability**. Check `./build-samples.sh help` to know more about it.
+    * `generateandtest` generate, and beside test samples like in Travis CI. Very interesting if you want check quickly know correctness of your modifications. Very interesting if you want check before push.
 
-    `list`: List all sample names
-    `generate`: Generate the sample if specified else generate all samples
-    `build` : Generate and test the sample if specified else generate and test all samples
-    `clean` : Clean the generated code for the sample if specified else clean all samples
+4. Then `./build-samples.sh startapplication sample_name` to start the application and check what is rendered in your web browser.
+
+#### Synopsis
+
+Command name can be as below:
+
+    `help`: display the help (recommended to read it).
+    `generate`: Generate sample(s). Not test are performed. You could see what is generated in samples/*-sample and open the project in your Text Editor/IDE.
+    `generateandtest` : Generate and test sample(s). Like `generate` + test as in Travis CI.
```

Comment:
I would call this command `verify` instead of `generateandtest`

Reply:
TODO